### PR TITLE
fix(post): fixed the profile picture feature for users liking and commenting on post

### LINKED ISSUE #2279: Profile images missing for 'Liked by' and 'Comment' users.
**Describe the bug**
Profile Images of users who have liked and commented is not visible.

**To Reproduce**

1. Open Individual Post screen
2. Like and Comment on the post.
3. Look at the profile images.

**Expected behavior**
Users respective profile images should be fetched and displayed in these places.

**Actual behavior**
Currently, there are blank circular avatars instead profile image.

**Screenshots**
talawa-issue-comment-image (1)

### PR DESCRIPTION
**Pull request by Abhisheksainii**
**What kind of change does this PR introduce?**

bugfix - Profile Images of users who have liked and commented is not visible.

**Issue Number:** #2279

Fixes #2279
1. Profile images of users liking and commenting on the post is visible now.

**Did you add tests for your changes?**

No, tests for services and models were already there.

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

Profile Images of users who have liked and commented on the post is not visible and was causing bad user experience.

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@Abhisheksainii Please fix the failing tests.

File: lib/views/after_auth_screens/feed/individual_post.dart [hunk 1]
```diff
@@ -193,7 +194,8 @@ class CommentTemplate extends StatelessWidget {
     return Row(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        const CircleAvatar(),
+        CircleAvatar(
+            backgroundImage: NetworkImage(comment.creator?.image ?? "")),
         Expanded(
```
Comment (SiddheshKukade) [inline lib/views/after_auth_screens/feed/individual_post.dart:199]: Please also add a fallback image to this as well

**Review by SiddheshKukade — CHANGES_REQUESTED** [review]
Please do that requested changes.

File: lib/views/after_auth_screens/feed/individual_post.dart (hunk 1 above)
Comment (Abhisheksainii) [reply lib/views/after_auth_screens/feed/individual_post.dart:199]: will do it, thanks!

**Comment by noman2002** [comment]
@Abhisheksainii Any update on this ?

**Comment by Cioppolo14** [comment]
@Abhisheksainii Are you working on this?

**Comment by Abhisheksainii** [comment]
> @Abhisheksainii Are you working on this?

Not right now

**Comment by palisadoes** [comment]
Closing due to inactivity

**Pull request closed without merging**